### PR TITLE
config to run GenXsecAnalyzer

### PR DESCRIPTION
**Pull request by kpedro88**
Running on TreeMaker file lists ensures consistency. Documentation is updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,16 @@ To check for new samples, use the above script [get_mcm.py](./Production/test/ge
 dasgoclient -query="dataset=/*/RunIISpring16MiniAOD*/MINIAODSIM"
 ```
 
+A convenient config to compute cross sections using the official GenXsecAnalyzer tool is provided, and can be used as follows:
```
Comment (aperloff) [inline README.md]: GenXsecAnalyzer --> GenXSecAnalyzer

File: README.md [hunk 2]
```diff
@@ -176,6 +176,16 @@ To check for new samples, use the above script [get_mcm.py](./Production/test/ge
...
+```
+cmsRun xsecfinder_cfg.py name=Autumn18.TTGamma_Dilept_TuneCP5_13TeV_madgraph_pythia8_ext1 numevents=1e7
```
Comment (aperloff) [inline README.md:181]: Do you want to mention the `redir` option?

File: Production/test/xsecfinder_cfg.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Read parameters
+from TreeMaker.Utils.CommandLineParams import CommandLineParams
+parameters = CommandLineParams()
+name = parameters.value("name","")
+redir = parameters.value("redir","root://cmsxrootd.fnal.gov/")
+numevents = int(parameters.value("numevents", 1e6))
+
+# handle site name usage
+if redir[0]=="T":
```
Comment (aperloff) [inline Production/test/xsecfinder_cfg.py:9]: This should be documented.

File: README.md (hunk 2 above)
Comment (kpedro88) [reply README.md:181]: Added to README

File: Production/test/xsecfinder_cfg.py (hunk 3 above)
Comment (kpedro88) [reply Production/test/xsecfinder_cfg.py:9]: Added to README

**Comment by aperloff** [comment]
Looks good to me.

**Pull request merged**